src/Components/Markdown.re

```diff
@@ -26,10 +27,19 @@ open {
            );
 
          let scope =
-           Oni_Extensions.LanguageInfo.getScopeFromLanguage(
-             languageInfo,
-             language,
-           );
+           switch (
+             Oni_Extensions.LanguageInfo.getScopeFromLanguage(
+               languageInfo,
+               language,
+             )
+           ) {
+           | Some(scope) as s => s
+           | None =>
```

Comment:
Minor tweak - you could use `OptionEx.or_` to remove the need for the switch statement.